### ISSUE TITLE
Resizing tilesheet, gameboard, 14x10 rectangle, whatever you want to call it...and more.

### ISSUE THREAD

**Issue by davidvorona** [Brethren-Studios/chimsockets#88]
There's more than enough room to add the UI on the x-axis, but there's currently not enough room for the chatbox, which will span across the bottom of the screen.

Removing the black and grey squiggly border might be enough; if it isn't, I might just change the size of each tile and keep it at 14x10. This isn't hard to readjust.

In other news, what do we think about zooming out more in the overworld? Like make the tiles smaller and the whole gameboard 21x15? Might make more sense if the overworld ends up being relatively expansive.

Would love to hear all your poignant thoughts on this subject.

**Comment by evanw555** [comment]
Let's call each game-oriented 14x10 grid a "cell", which is composed of currently 140 "tiles".

The cell dimensions are easily adjustable by making use of the constants file:

```
ROOM_DIMS: {
    HEIGHT: 10,
    WIDTH: 14
}
```
However, the background image (squiggly gray border) is a fixed image that relies on this size. The good news is that this is easy to recreate, so we shouldn't worry about this. The size of each tile, and the width of the border around the displayed cell window isn't fixed, they are defined as:

```
TILE_SIZE: 64,
PLAY_WINDOW_BORDER: 64,
```

We can easily reduce `PLAY_WINDOW_BORDER` to an arbitrary value to give us more space. Only other fixed value is the HTML canvas size. We may have to update the HTML to make that dependent on the constants file somehow.

As for reducing tile size, this is very easy.

For one, I did some experimenting and found that reducing the size of the entire play window is as simple as adding two lines in `graphics.js`:

- `stage.scaleX = .5;`
- `stage.scaleY = .5;`

This scales everything to half size. `stage` refers to the root object container in EaselJS. Scaling this container recursively scales every object and container within itself.

As for tiles themselves. The plan was to draw all the sprites at 32x32 to give the game a more pixelated, old-school feel. This works well with your idea; we can continue to draw everything at 32x32, and scale them to, say, 48x48 (1.5x). I can work on making this system as fluid as we would like it to be.

**In short,** this is all very flexible, and I think the game could benefit from having a larger grid. Biggest issue here is whether being so small might make it hard to see things and read things (since 32x32 itself feels too small).

**Comment by davidvorona** [comment]
What do we think about removing the squiggly grey border entirely? Canvas would be just the tiles the character can move on, and everything else on the page would be our React UI.

Obviously if we had a border, the current one would not be the final product, but I see no reason to have one. We need to use our space efficiently.

**Comment by evanw555** [comment]
We have discussed this enough. Issue closed.
